CONTRIBUTING.rst

```diff
@@ -33,34 +30,55 @@ You'll need to fork this repository and get the
 
 * Look for issues with the `good first issue
   <https://github.com/catalyst-cooperative/pudl/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue>`__
-  tag. These are issues that don't require a ton of PUDL-specific context, and
-  are relatively tightly scoped to boot.
+  tag in our `Community Kanban Board
+  <https://github.com/orgs/catalyst-cooperative/projects/9/views/19>`__. These
+  are issues that don't require a ton of PUDL-specific context, and are
+  relatively tightly scoped.
 
 * Comment on the issue and tag ``@com-dev`` (our Community Development Team) to
   let us know you're working on it. Feel free to ask any questions you might
   have!
 
 * Once you have an idea of how you want to tackle this issue, write out your
-  plan so we can guide you around obstacles in your way.
+  plan so we can guide you around obstacles in your way! Post a comment outlining:
+  * what steps have you broken this down into?
+  * what is the output of each step?
+  * how will one know that each step is working?
 
 **Work on it!**
 
-* Make a branch on your fork and open a draft PR early so we can discuss
-  concrete code! Please don't wait until it's all polished up - it's much easier
-  for us to help you when we can see the code evolve over time.
+* Make a branch on your fork and open a draft pull request (PR) early so we can
+  discuss concrete code! **Set the base branch to ``dev`` unless there's a good
+  reason otherwise.** Please don't wait until it's all polished up - it's much
+  easier for us to help you when we can see the code evolve over time.
 
 * Please make sure to write tests and documentation for your code - if you run
   into trouble with writing tests, let us know in the comments and we can help!
+  We automatically run the test suite for all PRs, but some of those will have
+  to be manually approved by Catalyst members for safety reasons.
 
-* Please try to keep your changes relatively small: stuff happens, and one's
+* **Try to keep your changes relatively small:** stuff happens, and one's
   bandwidth for volunteer work can fluctuate frequently. If you make a bunch of
   small changes, it's much easier to pause on a project without losing a ton of
-  context.
+  context. We try to keep PRs to **less than 500 lines of code.**
 
 **Get it merged in!**
 
 * Turn the draft PR into a normal PR and ping ``@com-dev``. We'll try to get
```

Comment:
Is this a github account associated with us? Or what do you mean here by pinging?

Reply:
I made a sub-team of catalyst called `@com-dev` which can be mentioned like @catalyst-cooperative/com-dev, which should have just dumped a notification into your GitHub inbox.

I can update the docs to be clear about this!